qml/FeedsPage.qml

```diff
@@ -258,6 +258,8 @@ BasePage {
                     }
                     onClicked:
                     {
+                        //revert check
+                        checked = Qt.binding( function () { return configEnabled; } );
```

Comment:
I don't understand what was the problem with the initial implementation to start with?